scripts/fortran_unit_testing/run_tests.py

```diff
@@ -180,6 +180,7 @@ def cmake_stage(name, test_spec_dir, build_optimized, use_mpiserial, mpirun_comm
             "cmake",
             "-C Macros.cmake",
             test_spec_dir,
+            "-DCIMEROOT="+_CIMEROOT,
```

Comment:
Why use an underscore here?  If this is intended to have the same value as the xml variable then shouldn't it have the same name as well?

Reply:
I may be confused about what you're asking here. This `_CIMEROOT` variable is pre-existing in this code; it looks like you introduced it in 00f113e86, where you set:

```python
_CIMEROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../..")
```

I assume you used a leading underscore to denote that it's private? Am I misunderstanding your question?

Reply:
No, that makes sense.  Thanks